x-pack/plugins/security_solution/common/license/policy_config.ts

```diff
@@ -74,6 +87,36 @@ export const isEndpointPolicyValidForLicense = (
   if (policy.windows.ransomware.supported !== defaults.windows.ransomware.supported) {
     return false;
   }
+  //
+  // only platinum or higher may enable memory_protection
+  if (policy.windows.memory_protection.mode !== defaults.windows.memory_protection.mode) {
```

Comment:
Is this duplicated?

Reply:
the comment is. the check is not :). I'll remove the comment.

Reply:
on a second look. The comment is also there for other protections. I am following the same pattern to keep it consistent.

Reply:
Maybe is a github UI error, but I'm seeing this condition duplicated:
```
if (policy.windows.memory_protection.mode !== defaults.windows.memory_protection.mode)
```

Reply:
ahh! I see it now :D thanks

Reply:
Fixed!